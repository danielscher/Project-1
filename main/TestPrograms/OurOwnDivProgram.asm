addiu $1, $0, 8
addiu $2, $0, 2
divu $1, $2
mfhi $3
mflo $4
