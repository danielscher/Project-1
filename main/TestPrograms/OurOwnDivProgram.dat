0x24010001
0x240201c8
0x0022001B
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x24010000
0x00001812
0x00002010
