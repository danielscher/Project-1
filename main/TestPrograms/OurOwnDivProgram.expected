@1 8
@2 2
@3 4
@4 0
@5 0
